Source/WebKit/UIProcess/Notifications/WebNotification.cpp

```diff
@@ -31,7 +31,7 @@
 
 namespace WebKit {
 
-WebNotification::WebNotification(const WebCore::NotificationData& data, WebPageProxyIdentifier pageIdentifier, const std::optional<WTF::UUID>& dataStoreIdentifier, IPC::Connection& sourceConnection)
+WebNotification::WebNotification(const WebCore::NotificationData& data, WebPageProxyIdentifier pageIdentifier, const std::optional<WTF::UUID>& dataStoreIdentifier, IPC::Connection* sourceConnection)
```

Comment:
I couldn't quite follow why IPC::Connection was changed from a reference to a pointer.

Reply:
There are now some notifications that don't come from *any* XPC service, so they have no source connection